### ISSUE TITLE
Look into Swedish and French translations

### ISSUE THREAD

**Issue by fnielsen** [fnielsen/afinn#9]
The commit by @mopinion contains some strange things: For instance "HÄNFÖRA 3" and English "Fed up  -3". This should be looked over and possibly fixed.

**Comment by richardohrvall** [comment]
Yes, quite a few words in the Swedish lexicon are in English. I saw another translation here: https://github.com/AlexGustafsson/sentiment-swedish/blob/develop/build/AFINN-sv-165.txt Maybe that could be a better option?

**Comment by mopinion** [comment]
That's probably the same one. A lot of English as well. I'm cleaning the Sv file
		
		

		Get Outlook for iOS
	




On Wed, Aug 23, 2017 at 11:13 PM +0200, "richardohrvall" <notifications@github.com> wrote:












Yes, quite a few words in the Swedish lexicon are in English. I saw another translation here: https://github.com/AlexGustafsson/sentiment-swedish/blob/develop/build/AFINN-sv-165.txt Maybe that could be a better option?

**Comment by richardohrvall** [comment]
Yes, I noticed that, that's why I deleted my comment. Let me know if you need any help with the translation.

**Comment by martinnordberg** [comment]
afinn = Afinn(language='sv')

Gives syntax error, while 'da' and 'en' works just fine.
